x-pack/test/api_integration/apis/management/rollup/rollup.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { API_BASE_PATH, ROLLUP_INDEX_NAME } from './constants';
+
+import { registerHelpers } from './rollup.test_helpers';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  const {
+    createIndexWithMappings,
+    getJobPayload,
+    createJob,
+    deleteJob,
+    startJob,
+    stopJob,
+    cleanUp,
+  } = registerHelpers({ supertest, es });
+
+  describe('Rollup jobs', () => {
+    after(() => cleanUp());
+
+    describe('Rollup indices', () => {
```

Comment:
How about using the endpoint path to describe the tests? I think that would help me understand what's being tested at a high-level without needing the read the tests themselves. So this would be "`${API_BASE_PATH}/indices`" and the next one would be "`${API_BASE_PATH}/index_pattern_validity `" and so on.

Reply:
That would expose the implementation detail and I am not sure I like it. What would happen with routes that are simply the root `/`? We'd have to make an exception. Aside, none of the other apps use this convention so we would need to change it everywhere.